app/javascript/chat/chat.jsx

```diff
@@ -1493,9 +1493,25 @@ export class Chat extends Component {
     const channelName = state.activeChannel
       ? state.activeChannel.channel_name
       : ' ';
+    const connectAnnouncement = (
+      <div className="bg-base-90 color-base-0 p-3 align-center fs-s m:fs-base lh-tight">
```

Comment:
Refactor (blocking): Add the [aria-live](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#live_regions) attribute and set it to polite, `aria-live="polite"`.